composer.json

```diff
@@ -16,7 +16,7 @@
 		"wmde/email-address": "~1.0",
 		"wmde/euro": "~1.0",
 		"wmde/clock": "~1.0",
-		"wmde/fundraising-donations": "~15.0",
+		"wmde/fundraising-donations": "dev-preselect-notification-opt-in",
```

Comment:
You can now set this to `~15.2`, I've merged our changes in the donation repo. Then You can regenerate the composer.lock file. If CI goes through, you can then merge this, Abban has accepted this PR